internal/host/validator.go

```diff
@@ -329,24 +343,34 @@ func (v *validator) printValidPlatform(c *validationContext, status ValidationSt
 	}
 }
 
-func (v *validator) getMemoryForRole(role models.HostRole) int64 {
+func (v *validator) getMemoryForRole(host *models.Host, cluster *common.Cluster, role models.HostRole) (int64, error) {
```

Comment:
`host` seems not to be used. Is it OK to remove it?

Reply:
That's correct. I'll remove it from the signature.